EvidenceApi/V1/Factories/ResponseFactory.cs

```diff
@@ -38,10 +38,11 @@ public static ResidentResponse ToResponse(this Resident domain, string? referenc
             this DocumentSubmission domain,
             DocumentType documentType,
             DocumentType? staffSelectedDocumentType = null,
-            Document? document = null
+            Document? document = null,
+            Claim? claim = null
         )
         {
-            return new DocumentSubmissionResponse()
+            return claim == null ? new DocumentSubmissionResponse()
```

Comment:
It would be good to update `ResponseFactoryTest` to cover this if statement

Reply:
👍